singlestoredb/fusion/handlers/sqrl.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import os
+
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+from singlestoredb.fusion.handler import SQLHandler
+from singlestoredb.fusion.result import FusionSQLResult
+
+class SqrlExplain(SQLHandler):
+    """
+    SQRL EXPLAIN string_value;
+
+    string_value = '<string-value>'
+
+    """
+    INKEEP_API_KEY = os.getenv('INKEEP_API_KEY')
+    INKEEP_INTEGRATION_ID = os.environ.get('INKEEP_INTEGRATION_ID')
+
+    def run(self, params: Dict[str, Any]) -> Optional[FusionSQLResult]:
```

Comment:
I'm wondering if the results should be returned as a result set rather than being printed. I don't know what the result will be used for, but printing is kind of a dead end since it just goes to stdout and it's hard to capture if needed.